src/main/resources/static/api.yml

```diff
@@ -1823,3 +2006,93 @@ components:
         - type
         - subType
         - note
+    Assessment:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        application:
+            $ref: '#/components/schemas/Application'
+        assignedToProbationOfficer:
```

Comment:
I wonder if an `assessor` property / schema would be clearer here than referring to `assignedToProbationOfficer`? 

And probably it would be better to avoid introducing a synonym for "allocate" here?

I'm thinking that the "assessment" has an "assessor" -- that's the person to whom the task of "assessment" has been "allocated"?

Reply:
Agreed on the synonym point!

On the probation officer part, assessors are a subet of probation officers though right?  So maybe `allocatedToProbationOfficer`?